script/assert_max_wasm_size.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echoerr() { echo "$@" 1>&2; }
+
+current_size=$(cat file_size)
+current_size="${current_size:0:$((${#current_size} - 1))}"
+
+max_size=2.1 # MB
+echo "Current size: ${current_size}M. expected maximum size: ${max_size}M"
```

Comment:
Does this line print `"Current size: 3M. expected maximum size: 2M" ? If so then:

1. These are not valid sentences. Please, take care of the quality of messages.
2. "M" is not a proper abbreviation for Megabyte.

Reply:
This is the format returned by `du -h`. I will rename it to `MiB`.